lib/node_modules/@stdlib/math/base/special/fresnel/benchmark/c/cephes/benchmark.c

```diff
@@ -0,0 +1,122 @@
+/**
+* Benchmark Cephes `fresnel`.
```

Comment:
`s/fresnel/fresnl/ since that's what cephes calls it, I suppose.

Reply:
Yes, you are correct. This comment should use the Cephes alias, but the printed output should match the package basename.